lib/rubysl/socket/rubysl.rb

```diff
@@ -27,11 +27,11 @@
 module RubySL
   module Socket
     def self.bsd_support?
-      Rubinius.bsd? || Rubinius.darwin?
+      Truffle::Support.bsd? || Truffle::Support.darwin?
```

Comment:
I think this would be nice as `Truffle::Platform.bsd?`

Reply:
Yes, again, we can move things into `Truffle` I just haven't done it yet.